.github/workflows/pr.yml

```diff
@@ -52,4 +52,4 @@ jobs:
         # when using macOS and python version 3.7
         echo "NO_PROXY='*'" >> $GITHUB_ENV
     - name: Run tests
-      run: pytest --strict-markers tests
+      run: pytest --strict-markers -W ignore::UserWarning tests
```

Comment:
@gruebel What are we ignoring here? 🤔

Reply:
yeah, there are custom warnings inside `detect-secrets` which are of course tested and they always show up as a warning inside the `pytest` run. Na value to see them, because they are intended to be there.

Reply:
these are the ones
```shell
tests/core/baseline_test.py::TestCreate::test_error_when_getting_git_tracked_files
  /home/runner/work/detect-secrets/detect-secrets/detect_secrets/core/scan.py:83: UserWarning: Did not detect git repository. Try scanning all files instead.
    log.warning('Did not detect git repository. Try scanning all files instead.')

tests/core/usage/filters_usage_test.py::test_disable_filter
  /home/runner/work/detect-secrets/detect-secrets/detect_secrets/core/usage/filters.py:192: UserWarning: Redundant --disable-filter "blah"
    log.warning(f'Redundant --disable-filter "{name}"')
```